src/handlers/transfers/handler.js

```diff
@@ -226,10 +219,13 @@ const fulfil = async (error, messages) => {
       histTimerEnd({ success: false, fspId: Config.INSTRUMENTATION_METRICS_LABELS.fspId })
       return true
     }
-    // Action Letter (AL) is used to denote breadcrumb flow end
-    const AL = action === TEA.COMMIT ? 'C' : (action === TEA.REJECT ? 'R' : (action === TEA.ABORT ? 'A' : '?'))
+    const CommitActionLetter = 'C'
```

Comment:
Should all of these not be put in an enum in the enum file?

Reply:
yes, it makes sense